config.json

```diff
@@ -101,7 +101,7 @@
           "intro-fn",
           "numbers"
         ],
-        "status": "wip"
+        "status": "beta"
```

Comment:
We could probably yolo this into active status now, but that's not essential.

Reply:
I went for it.